src/tools/python/templates/OpenTrackIO.html

```diff
@@ -71,10 +71,10 @@ <h2>Complete sample</h2>
             </div>
             
             <h2>Transport recommendations</h2>
-            <h3>UDP Multicast</h3>
```

Comment:
Since this section is no longer /only/ mentioning Multicast, shouldn't this title be just "UDP" ?